### PR TITLE
Allow overriding default.after_place_leaves and default.grow_sapling easily

### PR DESCRIPTION
**Pull request by SmallJoker**
Fix from p_gimeno to allow custom leafdecay, since overrriding `default.after_place_leaves` does not affect the leaves definition (changed reference).

### CODE REVIEW

File: mods/default/nodes.lua [hunk 1]
```diff
@@ -1663,7 +1668,6 @@ minetest.register_node("default:bush_sapling", {
 	paramtype = "light",
 	sunlight_propagates = true,
 	walkable = false,
-	on_timer = default.grow_sapling,
```
Comment (DS-Minetest) [inline mods/default/nodes.lua]: Why is this removed?

**Comment by paramat** [comment]
:+1:
